mobile/src/main/res/drawable/bg_empty_view.xml

```diff
@@ -0,0 +1,27 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I have added this illustration for onboard before, it is called ic_files_illustration so one of them should be deleted.